src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -182,7 +182,8 @@ function IOURequestStepAmount({
             const backendAmount = CurrencyUtils.convertToBackendAmount(Number.parseFloat(amount));
 
             if (shouldSkipConfirmation) {
-                if (iouType === CONST.IOU.TYPE.SPLIT) {
+                // Only skip confirmation when the split is not configurable, for now Smartscanned splits cannot be configured
+                if (iouType === CONST.IOU.TYPE.SPLIT && transaction?.iouRequestType === CONST.IOU.REQUEST_TYPE.SCAN) {
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/52386

On `MoneyRequestAmountForm`, the button text is because the `skipConfirmation` does not reflect the condition that IOU Manual Split can't use the skip confirmation.

We should update the flag `shouldSkipConfirmation` instead, so it can be reflected with the correct value in `MoneyRequestAmountForm`.